packages/host/app/services/matrix-service.ts

```diff
@@ -223,10 +229,28 @@ export default class MatrixService extends Service {
     }
   }
 
-  async startAndSetDisplayName(auth: LoginResponse, displayName: string) {
+  async initializeNewUser(auth: LoginResponse, displayName: string) {
+    displayName = displayName.trim();
+    let cardController = getOwner(this)!.lookup(
+      'controller:card',
+    ) as CardController;
+    cardController.workspaceChooserOpened = true;
```

Comment:
@jurgenwerk just FYI, this is a minor collision with your PR to replace the realm server home screen. If my PR gets merged first you'll need to resolve this. which i think just means deleting this altogether since the workspace chooser will be displayed after you first login from user registration

Reply:
Noted!